org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserProjectFileParser.java

```diff
@@ -21,20 +20,25 @@ public UserProjectFileParser() {
    * @throws UserFacingException
    * @throws UserInitiatedException
    */
-  public List<ProjectFile> parseFileJson(String json)
+  public void parseFileJson(
```

Comment:
I recommend creating a class for the return value here (instead of passing in lists that are filled in by this method).  This is more idiomatic and extensible and I think this class will be valuable -- it is a structured representation of all the assets in a student's workspace.

(Also, the @returns in the header comment should be updated.)

Reply:
made a follow-up PR to address these comments here! https://github.com/code-dot-org/java-ide/pull/28